<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html><head><title>Contribution of Syngenta Mutants to ABRC</title>
<link rel="STYLESHEET" href="seedgenes.css" type="text/css"></head><body>

<div class=header><div class=logoPos1>&nbsp;</div><div class=logoPos2>

<a href='http://www.okstate.edu'><img alt='OSU' SRC='images/osu.gif' border=0></a>&nbsp;
<a href='http://www.syngenta.com'><img alt='Syngenta' SRC='images/syngenta.gif' border=0></a>&nbsp;
<a href='http://www.vbi.vt.edu'><img alt='VBI' SRC='images/vbi.gif' border=0></a>
</div>
<a href='index.html'><img SRC='images/seedgenes.jpg' width='396' height='76' border=0></a>
</div>

<div class=menu1>
 <div class=menu2>
<a href="index.html">Home</a> | <a href="SeedGenesQuery.html">Database Query</a>
</div>
</div>

<div class="main700">
<h2>Contribution of Syngenta Mutants to ABRC</h2>

<p>
<table class=noborder>
<tr><td class=noborder><b>Seed Donors:</b>&nbsp;</td><td class=noborder>David W. Meinke (Oklahoma State University, Stillwater, OK)</td></tr>
<tr><td class=noborder>&nbsp;</td>  <td class=noborder>David A. Patton (Syngenta, Research Triangle Park, NC) </td></tr>
<tr></tr>
<tr><td class=noborder><b>Donation Date:</b>&nbsp;</td><td class=noborder>March 4, 2007</td></tr>
</table>

<p>
<a href="spreadsheets/Final_Corrected_ABRC_8R.xls">Complete Spreadsheet of Mutants (Excel)</a>
<p>
<div class="title">Background Information</div>

<p>
These seeds stocks (1412 recessive seed mutants of Arabidopsis) are being contributed to the Arabidopsis community following the conclusion of a collaborative project involving the Meinke and Patton laboratories that focused on a forward genetic screen of T-DNA insertion lines for recessive mutants with visible defects in seed development and/or seed pigmentation.  The T-DNA population screened was NOT the same as the SAIL/GARLIC population generated at TMRI and included in the SALK and TAIR insertion databases.  Please do not confuse these separate populations.  Many of the mutants included here are NOT tagged with T-DNA.  Other mutants are still unresolved with respect to tagging status.  These mutants are therefore NOT included in the SeedGenes database.  Because genetic complementation tests have not been performed with these mutants and the disrupted genes have not been identified, some of these mutants are likely to be allelic.  The 1412 mutants included in this donation therefore do NOT define 1412 distinct genes.  Seed stocks were obtained from heterozygous plants identified based on seed phenotype.  Approximately 2/3 of the progeny seeds should be heterozygotes.  The remainder should be wild-type for the recessive mutation.  Because we focused our attention on tagged mutants, terminal phenotypes noted here should be viewed as preliminary and need to be confirmed in future generations.

<p>The initial screening of insertion lines for defects in seed development is described in this publication:  
<div class='main'>
	<p>McElver et al. (2001)  Insertional mutagenesis of genes required for seed development in
Arabidopsis thaliana.  Genetics 159: 1751-1763.
<a href='publications/InsertionalMutagenesis_Genetics.pdf'>McElver et al. (2001)</a>
</div>
Information on genes included in the SeedGenes database is presented in this publication:
<div class='main'>
	<p>Tzafrir et al. (2004)  Identification of genes required for embryo development in Arabidopsis.
Plant Physiol. 135: 1206-1220.  
 <a href="publications/Tzafrir_etal_2004.pdf">Tzafrir et al. (2004)</a>
</div>
<p><em>Please cite one or both of these publications when making use of seeds included in this stock donation.</em>  

<p><div class=title>Overview of Mutant Collection</div>

<p><b>Overall Totals:</b>
<table class=abrc>
<col width=250px><col width=50px>
<tr><td>Embryo Defective</td><td class=textright>1268</td></tr>
<tr><td>Pigment Defective</td><td class=textright>140</td></tr>
<tr><td>Titan (Enlarged Nuclei) *</td><td class=textright>4</td></tr> 
</table>

<p><b>T-DNA Tagging Status:</b> 	 
<table class=abrc>
<col width=250px><col width=50px><col width=50px>
<tr><td>Not Tagged</td>     <td class=textright>873</td><td class=textright>N</td></tr>		
<tr><td>Possibly Tagged</td><td class=textright>27</td> <td class=textright>P</td></tr>
<tr><td>Unresolved</td>     <td class=textright>462</td><td class=textright>U</td></tr>
<tr><td>Not Determined</td> <td class=textright>50</td> <td class=textright>ND</td></tr>	
</table>

<p><b>Terminal Seed Phenotype: (Requires confirmation; Based on preliminary screening)</b>
<table class=abrc>
<col width=250px><col width=50px>
<tr><td>Zygotic </td><td class=textright>                         24</td></tr>
<tr><td>Zygotic/Preglobular</td><td class=textright>              34</td></tr>
<tr><td>Preglobular</td><td class=textright>			 248</td></tr>
<tr><td>Preglobular/Globular</td><td class=textright>		  83</td></tr>
<tr><td>Globular </td><td class=textright>			 367</td></tr>
<tr><td>Transition </td><td class=textright>			 149</td></tr>
<tr><td>Transition/Cotyledon</td><td class=textright>		   4</td></tr>
<tr><td>Cotyledon </td><td class=textright>			 246</td></tr>		
<tr><td>Variable </td><td class=textright>			  94</td></tr>
<tr><td>Albino 	</td><td class=textright>			  79</td></tr>
<tr><td>Pale Mature </td><td class=textright>			  61</td></tr>
<tr><td>Other 	</td><td class=textright>			  23</td></tr>
</table>
		 

<p><b>T-DNA Vector Used for Transformation: </b>
<table class=abrc>
<col width=250px><col width=50px>
<tr><td>CSA101LUC </td><td class=textright>  		 3</td></tr>		
<tr><td>CSA104 	 </td><td class=textright> 		489</td></tr>
<tr><td>CSA104LUC </td><td class=textright>  		268</td></tr>
<tr><td>CSA110 	 </td><td class=textright> 		201</td></tr>
<tr><td>DAP101 	  </td><td class=textright>		354</td></tr>
<tr><td>RW1 	</td><td class=textright>		63</td></tr>
<tr><td>SKI15 	</td><td class=textright>		34</td></tr>	
</table>

<p><b>Selection Agent for Identifying Resistant Plants: </b>
<table class=abrc>
<col width=250px><col width=50px>
<tr><td>Basta 	</td><td class=textright>  1349</td></tr>	
<tr><td>Hygromycin </td><td class=textright> 63</td></tr>	
</table>

<p><b>Parental Ecotype: </b>
<table class=abrc>
<col width=250px><col width=50px>
<tr><td>Columbia </td><td class=textright>1320</td></tr>		
<tr><td>WS </td><td class=textright>        92</td></tr>	
</table>

<p><b>Parental Genotype:</b>
<table class=abrc>
<col width=250px><col width=50px>
<tr><td>Wild-type </td><td class=textright>1211</td></tr>		
<tr><td>Quartet </td><td class=textright> 201</td></tr>	
</table>

<p>Footnotes:  

<p>* Anyone interested in these 4 lines should contact the Meinke laboratory before seed
distribution in order to discuss potential uncertainties about these stocks.  

<p>Line numbers followed by ".1" or ".2" were originally part of a double mutant that was separated into single mutants in subsequent generations.  

<p><div class=title>Additional Information</div>

<p>Tagging status was determined by screening resistant transplants for the presence of defective seeds.  Details are presented in McElver et al. (2001).  Information on parental resistant : sensitive (R:S) ratios, which reflects the number of active inserts present in each parental line, is included with each seed stock.  With known tagged mutants (not included in this collection), all resistant transplants were found to be heterozygous for the mutation, consistent with perfect linkage between the T-DNA insert and the seed phenotype.  With untagged mutants, lines with a single functional insert yielded a mixture of wild-type and heterozygous plants when screened for the seed phenotype.  Mutants classified as possibly tagged yielded a low frequency of resistant wild-type plants, a result that could still be consistent with tagging.  Mutants classified as unresolved with respect to tagging status typically contained multiple inserts that interfered with the ability to resolve tagging status through screening of resistant transplants.  Based on previous experience with similar insertion lines, we anticipate that about 30% of these "unresolved" mutants are likely to be tagged.  

<div class='footer'>
Address comments to David Meinke, Project PI <a href="mailto:meinke@okstate.edu" class='footer_link'>(meinke@okstate.edu)</a>
</div>


</body></html>
