<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html><head><title>New Arabidopsis 2010 Project</title>
<link rel="STYLESHEET" href="seedgenes.css" type="text/css"></head><body>

<div class=header>
<div class=logoPos1>&nbsp;</div>
<div class=logoPos2>
<a href='http://www.okstate.edu'><img alt='OSU' SRC='images/osu.gif' border=0></a>&nbsp;
<a href='http://www.syngenta.com'><img alt='Syngenta' SRC='images/syngenta.gif' border=0></a>&nbsp;
<a href='http://www.vbi.vt.edu'><img alt='vbi' SRC='images/vbi.gif' border=0></a>
</div>
<a href='index.html'><img SRC='images/seedgenes.jpg' width='396' height='76' border=0></a></div>

<div class=menu1>
 <div class=menu2>
<a href="index.html">Home</a> | <a href="AccessPage.html">Access Page</a> | <a href="SeedGenesQuery.html">Database Query</a>
</div>
</div>

<div class="main">
<h1>New Arabidopsis 2010 Project</h1>


<table class=label>
<tr><td>Title: </td><td> Saturating the Arabidopsis Knockout Collection with Lethals</td></tr>
<tr><td>PI: </td><td>  David Meinke, Oklahoma State University</td></tr>
<tr><td>Co-PI:</td><td>    Allan Dickerman, Virginia Bioinformatics Institute </td></tr>
<tr><td>NSF Funding: </td><td> 2010 Program, Division of Biological Infrastructure (DBI)</td></tr>
<tr><td>Funding Period: </td><td>September 2006 - August 2009 </td></tr>
<tr><td>Program Manager:&nbsp;</td><td>Parag Chitnis</td></tr>
</table>
<p>
<span class='label'>Abstract:</span>
<br>
One goal of the Arabidopsis research community is to identify a knockout for every annotated gene. Missing from the current collection of homozygous knockout mutants are the essential genes for which mutant seeds abort or fail to form because of gametophytic lethality. The aim of this project is to coordinate efforts to fill this gap in the knockout collection. The result will be a valuable resource for future experiments and benchmark information on the consequences of disrupting non-redundant, essential genes in Arabidopsis. The principal investigators have already established a robust database (www.seedgenes.org) of 300 EMB genes required for seed development in Arabidopsis. Three approaches are planned to expand upon this community resource. The first involves genes where the existing knock-out project finds insertion lines with confirmed heterozygotes but no homozygous knockouts. This list will be filtered to identify genes that represent promising candidates for embryo or gametophytic lethals based on known sequence redundancy, orthology, expression patterns, and shared features with known essential genes. Selected lines will then be planted, genotyped, and confirmed. They will be screened for defects in seed and gametophyte development. The goal is to analyze 450 to 600 promising insertion lines over three years. The second approach involves collecting information on lethal mutants identified in other laboratories. This approach has worked well for seed mutants and will expand in the future to include putative gametophytic mutants. The third approach involves identification of promising candidates for essential genes among those remaining genes for which insertions have not been found or cannot be confirmed. This approach will generate a list of candidate genes that are required for both male and female gametophyte development. Results of these combined approaches will be incorporated into the SeedGenes and TAIR databases. Genes with unknown functions and without orthologs in other model organisms but with a confirmed seed phenotype will be examined in most detail. These efforts will enhance our understanding of the unknowns in Arabidopsis by focusing on genes known to be essential. 
<p>
The broader educational impact of this project includes participation of high school students in research activities and outreach programs on Arabidopsis mutants and plant molecular genetics involving regional high schools, the Extended Studies Program at Stillwater Junior High School, and the Oklahoma School for Science and Mathematics in conjunction with the OSU Plant Biotechnology Network. These programs were successfully initiated during the previous grant period. 

<H1>Original Project Objectives and Timetable</H1>

<font size=6><b>T</b></font>he purpose of the Arabidopsis SeedGenes
		      Project is to coordinate the collection,
		      analysis, and presentation of information on
		      essential, non-redundant genes that give a seed
		      phenotype when disrupted by
		      mutation. Arabidopsis appears to contain ~750
		      such EMB genes required for seed development and
		      another ~200 PDE genes required for normal seed
		      pigmentation. Our long-term goal is to help the
		      community establish a complete collection of
		      Arabidopsis genes with a mutant seed
		      phenotype. This information is needed to focus
		      attention on genes with important cellular
		      functions and to assess from a genetic
		      perspective the extent of functional redundancy
		      in the Arabidopsis genome.
 
<p>

<b>Specific Project Objectives:</b>
<ul>
<li>Present synthesized information on 500 mutants defective in 
375 different genes required for seed development (300 EMB; 
75 PDE) in a web-accessible database.

<li>Identify duplicate alleles for many of these genes through 
a combination of forward and reverse genetics.  

<li>Summarize expression data on genes required for normal seed development.    

<li>Provide distribution quantities of mutant seed stocks to ABRC.

<li>Establish a model for public release of private sector resources.

<li>Develop outreach and training programs that complement research 
efforts in Arabidopsis functional genomics.  
</ul>
<p>The SeedGenes Project database is organized into two major
		      sections. One section deals with genes and the
		      other with mutant alleles. The database can be
		      queried for detailed information on a single
		      gene to create a SeedGenes Profile. Queries can
		      also generate lists of genes or mutants that fit
		      specified criteria. The fifth database release
		      (May 2004) contains information on 260 essential
		      genes and 408 mutants. Included are many genes
		      identified at Syngenta and all known genes
		      described in the literature. Anticipated future
		      releases are summarized in the following table
		      from the original proposal: 
</p>
<p>
<b>Target Numbers of Essential Gene Profiles In Database:</b>
</p>
<div style='margin-left: 2em;'>
<TABLE WIDTH=388 BORDER=1 BORDERCOLOR="#000000" CELLPADDING=1 CELLSPACING=0>
	<COL WIDTH=51>
	<COL WIDTH=42>
	<COL WIDTH=57>
	<COL WIDTH=67>
	<COL WIDTH=46>
	<COL WIDTH=61>
	<COL WIDTH=48>
	<TR VALIGN=TOP>
		<TD ALIGN=CENTER>Month</TD>
		<td ALIGN=CENTER>Year</TD>
		<td ALIGN=CENTER>Project Months</TD>
		<td ALIGN=CENTER>Syngenta EMB</TD>

		<td ALIGN=CENTER>Other EMB</TD>

		<td ALIGN=CENTER>Seed Pigment</TD>
		<td ALIGN=CENTER>Total Genes</TD>
	</TR>
	<TR VALIGN=TOP>
		<td ALIGN=CENTER>Sept</TD>
		<td ALIGN=CENTER>2001</TD>
		<td ALIGN=CENTER>0</TD>
		<td ALIGN=CENTER>-</TD>
		<td ALIGN=CENTER>-</TD>
		<td ALIGN=CENTER>-</TD>
		<td ALIGN=CENTER>-</TD>
	</TR>
	<TR VALIGN=TOP>
		<td ALIGN=CENTER>March</TD>
		<td ALIGN=CENTER>2002</TD>
		<td ALIGN=CENTER>6</TD>
		<td ALIGN=CENTER>50</TD>
		<td ALIGN=CENTER>25</TD>
		<td ALIGN=CENTER>25</TD>
		<td ALIGN=CENTER>100</TD>
	</TR>
	<TR VALIGN=TOP>
		<td ALIGN=CENTER>Sept</TD>
		<td ALIGN=CENTER>2002</TD>
		<td ALIGN=CENTER>12</TD>
		<td ALIGN=CENTER>100</TD>
		<td ALIGN=CENTER>40</TD>
		<td ALIGN=CENTER>30</TD>
		<td ALIGN=CENTER>170</TD>
	</TR>
	<TR VALIGN=TOP>
		<td ALIGN=CENTER>March</TD>
		<td ALIGN=CENTER>2003</TD>
		<td ALIGN=CENTER>18</TD>
		<td ALIGN=CENTER>150</TD>
		<td ALIGN=CENTER>50</TD>
		<td ALIGN=CENTER>40</TD>
		<td ALIGN=CENTER>240</TD>
	</TR>
	<TR VALIGN=TOP>
		<td ALIGN=CENTER>Sept</TD>
		<td ALIGN=CENTER>2003</TD>
		<td ALIGN=CENTER>24</TD>
		<td ALIGN=CENTER>165</TD>
		<td ALIGN=CENTER>65</TD>
		<td ALIGN=CENTER>50</TD>
		<td ALIGN=CENTER>280</TD>
	</TR>
	<TR VALIGN=TOP>
		<td ALIGN=CENTER>March</TD>
		<td ALIGN=CENTER>2004</TD>
		<td ALIGN=CENTER>30</TD>
		<td ALIGN=CENTER>180</TD>
		<td ALIGN=CENTER>80</TD>
		<td ALIGN=CENTER>60</TD>
		<td ALIGN=CENTER>320</TD>
	</TR>
	<TR VALIGN=TOP>
		<td ALIGN=CENTER>Sept</TD>
		<td ALIGN=CENTER>2004</TD>
		<td ALIGN=CENTER>36</TD>
		<td ALIGN=CENTER>190</TD>
		<td ALIGN=CENTER>90</TD>
		<td ALIGN=CENTER>65</TD>
		<td ALIGN=CENTER>345</TD>
	</TR>
	<TR VALIGN=TOP>

		<td ALIGN=CENTER>March</TD>
		<td ALIGN=CENTER>2005</TD>
		<td ALIGN=CENTER>42</TD>
		<td ALIGN=CENTER>195</TD>
		<td ALIGN=CENTER>95</TD>
		<td ALIGN=CENTER>70</TD>
		<td ALIGN=CENTER>360</TD>
	</TR>
	<TR VALIGN=TOP>
		<td ALIGN=CENTER>Sept</TD>
		<td ALIGN=CENTER>2005</TD>
		<td ALIGN=CENTER>48</TD>
		<TD ALIGN=CENTER>200</TD>
		<TD ALIGN=CENTER>100</TD>
		<TD ALIGN=CENTER>75</TD>
		<TD ALIGN=CENTER>375</TD>
	</TR>
</TABLE>
<dl>
<dt>Syngenta EMB:
<dd>Cloned genes from Syngenta mutant collection

<dt>Other EMB:
<dd>Cloned genes from community

<dt>Seed Pigment:
<dd>Additional genes from Syngenta and community

<dt>Total Genes:
<dd>Cumulative total of genes identified
</dl>
</div>




</div>
</body></html>
