<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html><head><title>Explanation of Flanking Sequence Information</title>
<link rel="STYLESHEET" href="seedgenes.css" type="text/css"></head><body>

<div class=header><div class=logoPos1>&nbsp;</div><div class=logoPos2>

<a href='http://www.okstate.edu'><img alt='OSU' SRC='images/osu.gif' border=0></a>&nbsp;
<a href='http://www.syngenta.com'><img alt='Syngenta' SRC='images/syngenta.gif' border=0></a>&nbsp;
<a href='http://www.vbi.vt.edu'><img alt='VBI' SRC='images/vbi.gif' border=0></a>
</div>
<a href='index.html'><img SRC='images/seedgenes.jpg' width='396' height='76' border=0></a>
</div>
<div class=menu1>
 <div class=menu2>
<a href="index.html">Home</a> | <a href="AccessPage.html">Access Page</a> | <a href="SeedGenesQuery.html">Database Query</a>
</div>
</div>

<div class=main700>
<h1>Explanation of Flanking Sequence Information</h1>

<p>This document presents additional details on predicted T-DNA insertion sites for mutants obtained from the Syngenta, Salk, Genoplante, and GABI collections.  The following information is presented in the HTML document linked to each mutant with flanking sequence available:  
<p>
<ul>
<li>Locus and line number for each insertion mutant 
<li>TAIR Sequence View (TIGR 5.0; 11/2004) with triangles marking insertion points 
<li>The first 15 bp perfect match of each flanking sequence to the genome highlighted  
<li>Actual flanking sequences obtained for each border (Border 1 = Most upstream)  
<li>BLASTN results (WU-BLAST2 at TAIR) used to locate borders in the genome 
<li>Summary information on predicted insertion sites 
</ul>

<i>The following notes may be helpful in understanding the information presented:</i>
<p>
<ul>
<li>Insertion sites are defined as the start of the first 15 bp perfect alignment between the sequenced border and the Arabidopsis genome.  
<li>Insertion sites are marked with a triangle adjacent to a boxed sequence in the SeqView.
<li>Red triangles are based on border sequences from Syngenta, Salk, Genoplante, or GABI. 
<li>Green triangles are based on confirmation sequences from the Meinke laboratory and the Oklahoma State University DNA/Protein Resource Facility.  
<li>When the BLAST alignment begins with a 15 bp perfect match, a single triangle is placed adjacent to the boxed region.  
<li>When the BLAST alignment does not begin with a perfect match, a second triangle indicates the point of initial alignment.  
<li>The location of the first 15 bp perfect alignment is highlighted in the BLAST output.  
<li>Clicking on each highlighted sequence links you to the next occurrence of that sequence.   
<li>Each Sequence View is oriented with the ATG of the gene in question at the top.  Border sequences from the complementary strand (minus/plus in BLAST output) must therefore be read in the opposite direction.  
<li>Borders that include sequences upstream or downstream of the coding region often required alignment with BAC sequences rather than with gene sequences.  
<li>Insertion points are calculated in relation to the "A" in "ATG" or the final letter of the stop codon for downstream insertions.  
<li>Consistent with TAIR nomenclature, introns can be located in 5'and 3' UTRs.  The first exon may therefore not contain the ATG.
</ul>
</div>

<div class='footer'>
Address comments to David Meinke, Project PI <a href="mailto:meinke@okstate.edu" class='footer_link'>(meinke@okstate.edu)</a>
</div>

</body></html>
