<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML><HEAD>
<TITLE>SeedGenes Glossary</TITLE>
<link rel='STYLESHEET' href='seedgenes.css' type='text/css'>
</HEAD>
<BODY>
<div class=header><div class=logoPos1>&nbsp;</div><div class=logoPos2>
<a href='http://www.okstate.edu'><img alt='OSU' SRC='images/osu.gif' border=0></a>&nbsp;
<a href='http://www.syngenta.com'><img alt='Syngenta' SRC='images/syngenta.gif' border=0></a>&nbsp;
<a href='http://www.vbi.vt.edu'><img alt='VBI' SRC='images/vbi.gif' border=0></a>
</div><a href='index.html'><img SRC='images/seedgenes.jpg' width='396' height='76' border=0></a></div>
<div class=menu1><div class=menu2>
<a href='index.html'>Home</a> | <a href='AccessPage.html'>Access Page</a> | <a href='SeedGenesQuery.html'>Database Query</a>
</div></div>
<div class=main700>
<h1>SeedGenes Glossary</h1>
<div class=title>Terms in SeedGenes Query Form</div>
<dl>
<dt class=g><a class='glossary' name="Gene_Class_Query">Gene Class</a></dt>
<dd>Mutations in genes classified as "Embryo Defective" give an embryo-defective phenotype characterized by a morphological defect in seed development.  Those classified as "Seed Pigment" result in a defect in seed pigmentation (albino, pale green, fusca) but not in morphogenesis.  With the "50% Defective Seeds" class, siliques produce ~50% mutant seeds following self-pollination of heterozygotes, regardless of pollen genotype.
</dd>
<dt class=g><a class='glossary' name="Chromosome">Chromosome</a></dt>
<dd>Select the chromosome number to limit your search to genes located in that region.
</dd>
<dt class=g><a class='glossary' name="Gene_Identity_Confidence">Gene Identity Confidence</a></dt>
<dd>Evidence that the seed phenotype observed is caused by disruption of the
  gene indicated.  Confirmed: Second mutant allele found disrupted in the same
  gene or mutant phenotype rescued by introduction of the wild-type gene;  Not
  Confirmed: Single mutant allele with sequence information available but no
  reason to doubt gene identity.
<!--Uncertain: Insertion or mutation site not within coding region or 5' UTR  and either downstream of stop codon or more than 250 bp upstream of start codon;  Questionable: Contradictory evidence from different sources raises questions about accuracy of gene identity.-->
</dd>
<dt class=g><a class='glossary' name="Second_Allele_Available">Second Allele Available</a></dt>
<dd>Select "Yes" to limit your search to genes for which a second mutant allele is available, either in the SeedGenes database or elsewhere in the community.
</dd>
<dt class=g><a class='glossary' name="FLcDNA_Sequenced">FLcDNA Sequenced</a></dt>
<dd>Select "Yes" to limit your search to genes for which a full-length cDNA has been sequenced.
</dd>
<dt class=g><a class='glossary' name="Protein_Function_Keyword">Protein Function Keyword</a></dt>
<dd>Enter here a keyword (or string of up to 3 words) that may be included in the brief description of predicted functions for genes in the SeedGenes database.  Use "Unknown" or "Uncertain" for genes with questionable functions.  Examples:  Ribosomal protein; PPR Protein; Syntaxin; Vitamin C; tRNA synthetase; Cohesin; mTERF; Dynamin; Ankyrin Repeat; Translation; Transcription.
</dd>
<dt class=g><a class='glossary' name="Mutant_Class_Query">Mutant Class</a></dt>
<dd>Mutant alleles classified as "Embryo Defective" give an embryo-defective phenotype characterized by a morphological defect in seed development.  Those classified as "Seed Pigment" result in a defect in seed pigmentation (albino, pale green, fusca) but not in morphogenesis.  With the "50% Defective Seeds" class, siliques produce ~50% mutant seeds following self-pollination of heterozygotes, regardless of pollen genotype.
</dd>
<dt class=g><a class='glossary' name="Original_Seed_Source">Original Seed Source</a></dt>
<dd>Source of mutant seeds.  The Syngenta collection of mutants described here was generated at Research Triangle Park, NC and differs from the population designed for reverse genetics at Torrey Mesa Research Institute in San Diego, CA.  The INRA/Genoplante collection of T-DNA insertion lines was generated at Versailles, France.  Information on the Salk collection of insertion lines can be obtained at www.signal.salk.edu.  
</dd>
<dt class=g><a class='glossary' name="Terminal_Phenotype_q">Terminal Phenotype</a></dt>
<dd>Characteristic terminal stage of development reached by mutant embryos prior to desiccation.  The assigned class corresponds to the one most frequently observed or the most developmentally advanced class found in at least 10% of the seeds examined.  Seeds classified as preglobular often contain an early globular embryo too small to be seen upon seed dissection.  Pigment Defective includes albino, pale mature, and fusca mutants.  Click on "Drawings" to query mutants by terminal phenotype subclass.
</dd>
<dt class=g><a class='glossary' name="Special_Feature">Special Feature</a></dt>
<dd>Distinctive morphological abnormalities frequently observed in mutant seeds.
</dd>
<dt class=g><a class='glossary' name="Nomarski_Images">Nomarski Images</a></dt>
<dd>Refers to the availability of Nomarski (DIC) images of cleared mutant seeds.
</dd>
<dt class=g><a class='glossary' name="Mutagen_Type">Mutagen Type</a></dt>
<dd>Mutagen used to induce the mutation.
</dd>
<dt class=g><a class='glossary' name="Ecotype_Query">Ecotype</a></dt>
<dd>Name of the <i>Arabidopsis</i> ecotype in which the mutation was generated.
</dd>
<dt class=g><a class='glossary' name="Tagged_Insertion_Mutant">Tagged Insertion Mutant</a></dt>
<dd>Select "Yes" to limit your search to mutants in which the essential gene is tagged with T-DNA or a transposable element.
</dd>
<dt class=g><a class='glossary' name="Border_Recovery_Rank">Border Recovery Rank</a></dt>
<dd>These ratings reflect the level of certainty that the correct insertion site has been identified.  Refer to <a href='publications/InsertionalMutagenesis_Genetics.pdf'>McElver <i>et al</i>. (2001) Genetics 159: 1751-1763.</a> for a complete definition and discussion.  Use this option to search Syngenta, Salk, and INRA/Genoplante mutants.  Rank A = Flanking DNA recovered from both sides of the insert and found to identify the same gene region.  Rank B = Flanking DNA recovered from a single side of the insert.  Recovery of the second border either failed or was not attempted.  Rank Cs = Flanking DNA recovered from both sides of an insert associated with deletion or potential disruption of two or more adjacent genes.  Rank C = Flanking DNA recovered from 2 different inserts located at some distance apart on the same chromosome.  The second insert escaped detection with genetic analysis.  Rank C and Cs alleles are included only if supported by results of genetic complementation tests.
</dd>
<dt class=g><a class='glossary' name="Initial_Ratio_R_S_Seedlings">Initial Ratio R:S Seedlings</a></dt>
<dd>Refer to <a href='publications/InsertionalMutagenesis_Genetics.pdf'>McElver <i>et al</i>. (2001) Genetics 159: 1751-1763.</a> for a complete definition and discussion of ratio classes for mutants generated at Syngenta.  Parental insertion lines with a "Low" ratio (&lt;&nbsp;5:1) of resistant (R) to sensitive (S) progeny typically contained a single site of insertion; those with "Moderate" (5.1-25:1) and "High" (&gt;&nbsp;25:1) ratios initially contained additional unlinked inserts that were later removed through random assortment.
</dd>
<dt class=g><a class='glossary' name="Aberrant_Segregation">Aberrant Segregation</a></dt>
<dd>Select "Yes" to limit your search to mutants that do not exhibit the expected 25% mutant seeds following self-pollination of heterozygotes.  Cutoff for inclusion: Chi-square > 11.0; (P < 0.001).  
</dd>
<dt class=g><a class='glossary' name="Aberrant_Distribution">Aberrant Distribution</a></dt>
<dd>Select "Yes" to limit your search to mutants that exhibit a non-random distribution of mutant seeds along the length of the silique.  Such deviations often indicate an effect of the mutant allele on pollen development or pollen-tube growth.  Cutoff for inclusion: Chi-square > 11.0; (P < 0.001).  
</dd>
<dt class=g><a class='glossary' name="Seed_Color_q">Seed Color</a></dt>
<dd>Average color of mutant seed prior to desiccation at maturity.
</dd>
<dt class=g><a class='glossary' name="Embryo_Color">Embryo Color</a></dt>
<dd>Average color of mutant embryo prior to desiccation at maturity.
</dd>

</dl><br>
<div class=title>Terms in SeedGenes Gene Profile</div>
<dl>
<dt class=g><a class='glossary' name="Gene_Symbol">Gene Symbol</a></dt>
<dd>Standard genetic symbol for an <i>Arabidopsis</i> gene identified by mutation.  Refer to <a href="http://www.arabidopsis.org">www.arabidopsis.org</a> for full names.  The wild-type allele is written in CAPS; mutant alleles are lower case.  A space has been added between letters of a gene symbol (ABC) and numbers that designate different loci (ABC 1 and ABC 2) to minimize confusion between numbers and letters.  The number 1 is optional if a single locus is known (i.e. ABC = ABC 1).  In cases where a gene is known by more than one symbol, the original or established name is given here, with others listed in the Alias section.
</dd>
<dt class=g><a class='glossary' name="Chromosome_Locus">Chromosome Locus</a></dt>
<dd>Chromosome locus identifier for an Arabidopsis gene predicted from the sequenced genome.  The chromosome number (1,2,3,4,5) is followed by gene (g) numbers starting at the top of each chromosome.  Assignments shown here are from TAIR (<a href='http://www.arabidopsis.org'>www.arabidopsis.org</a>).  
</dd>
<dt class=g><a class='glossary' name="Alleles_in_Database">Alleles in Database</a></dt>
<dd>Number of different mutant alleles included in the SeedGenes database.  Mutant alleles were chosen for inclusion based on seed availability and level of phenotypic and/or molecular characterization.
</dd>
<dt class=g><a class='glossary' name="Identity_Confidence">Identity Confidence</a></dt>
<dd>Evidence that the seed phenotype observed is caused by disruption of the gene indicated.  Confirmed: Second mutant allele found disrupted in the same gene or mutant phenotype rescued by introduction of the wild-type gene;  Not Confirmed: Single mutant allele with sequence information available but no reason to doubt gene identity;  Uncertain: Insertion or mutation site not within coding region or 5' UTR  and either downstream of stop codon or more than 250 bp upstream of start codon;  Questionable: Contradictory evidence from different sources raises questions about accuracy of gene identity.
</dd>
<dt class=g><a class='glossary' name="Alias_Symbol">Alias Symbol</a></dt>
<dd>Alternate genetic symbol for the gene described.  Includes the laboratory that identified or named the alias locus and evidence for identity between the gene represented by the alias and the standard genetic symbol.  
</dd>
<dt class=g><a class='glossary' name="Gene_Class">Gene Class</a></dt>
<dd>Mutations in genes classified as "Embryo Defective" give an embryo-defective phenotype characterized by a morphological defect in seed development.  Those classified as "Seed Pigment" result in a defect in seed pigmentation (albino, pale green, fusca) but not in morphogenesis.  With the "50% Defective Seeds" class, siliques produce ~50% mutant seeds following self-pollination of heterozygotes, regardless of pollen genotype.
</dd>
<dt class=g><a class='glossary' name="Predicted_Function">Predicted Function</a></dt>
<dd>Predicted function of the essential gene based on sequence comparison and/or experimental data.  Users are encouraged to contact the central <i>Arabidopsis</i> database (<a href='http://www.arabidopsis.org'>www.arabidopsis.org</a>) for more details.  Please contact the <b>SeedGenes Project</b> with corrections and updated information.
</dd>
<dt class=g><a class='glossary' name="Function_Details">Function Details</a></dt>
<dd>Additional comments about proposed cellular function of the essential gene.
</dd>
<dt class=g><a class='glossary' name="Gene_Evidence">Gene Evidence</a></dt>
<dd>Evidence that disruption of the gene described here results in the mutant phenotype observed.  Information on different mutant alleles in the SeedGenes database is included.   "Molecular Complementation" indicates that the mutant phenotype was rescued by transformation with a wild-type copy of the gene.  "Multigene Disruption" includes deletions that cover several adjacent genes.  "Phenotypic Reversion" reflects the presence of revertant sectors formed by transposon excision.  "Double Border Recovery" refers to insertion mutants where both sides of the insert map to the same gene region.
</dd>
<dt class=g><a class='glossary' name="Evidence_Source">Evidence Source</a></dt>
<dd>Laboratory that identified the gene disrupted in the mutant described.
</dd>
<dt class=g><a class='glossary' name="Database_Alleles">Database Alleles</a></dt>
<dd>Number of alleles in the SeedGenes database with the listed evidence and source.
</dd>
<dt class=g><a class='glossary' name="Database_Release">Database Release</a></dt>
<dd>Date this gene was first included in the SeedGenes project.  
</dd>
<dt class=g><a class='glossary' name="Predicted_Sequences">Predicted Sequences</a></dt>
<dd>Gene:  Length in base pairs of gene predicted from current gene model.  Base #1 begins with the first base of the gene model, which in some cases corresponds to the first AUG used in translation and in other cases includes the 5' UTR present in FLcDNAs.  RNA: Length in bases of processed RNA derived from predicted gene.  This may include 5' and 3' UTRs but not predicted or confirmed introns.  Protein: Length in amino acids of predicted protein product.  For updated information, refer to TAIR (<a href='http://www.arabidopsis.org'>www.arabidopsis.org</a>).
</dd>
<dt class=g><a class='glossary' name="cDNA_Status">cDNA Status</a></dt>
<dd>Full-length (FL) cDNA is noted if sequenced.  Existence of an EST is noted when a FLcDNA is not available.  Accession numbers for FLcDNAs are presented when available.
</dd>
<dt class=g><a class='glossary' name="BLAST_Results">BLAST Results</a></dt>
<dd>Predicted protein sequences presented in the SeedGenes Profile were used for three different types of BLAST analysis.
<p><u><i>BLASTP x Arabidopsis Proteins</i></u>:  This analysis was used to
  search for Arabidopsis proteins with similar sequences and potentially
  related functions.  All protein hits with an E value of less than 1E-60 are
  included in the table.  The database queried was the TAIR9_pep_20090619
  sequence set from TAIR
  (<a href='http://www.arabidopsis.org'>www.arabidopsis.org</a>). 
<p>
<u><i>TBLASTN x Plant ESTs</i></u>: This analysis was used to determine the
taxonomic distribution of homologs of SeedGenes entries across a diversity of
plants.  The datasets searched against are the assembled EST "uni-gene" sets
for 28 plant species compiled by
the <a href='http://compbio.dfci.harvard.edu/tgi/plant.html'>Gene Index
  Project</a> at Harvard.  
Using assembled gene indices provides a much less redundant view of the gene
content than does use of raw ESTs.  The number of strong hits (E-value
<= 1E-30) to each species is shown.
<p><u><i>BLASTP x Non-Plant Model Systems</i></u>:  This analysis was used to search for proteins with similar sequences in each of 9 different model organisms ranging from human to bacteria.  The number of hits (E-value <= 1E-30) in each species is reported.
</dd>
<dt class=g><a class='glossary' name="Subcellular_Localization">Subcellular Localization</a></dt>
<dd>Two methods are used to computationally predict where the protein will be
  localized within the cell: TargetP (see below) and SUBA or
  <a href='http://suba.plantenergy.uwa.edu.au/'>SUB-cellular location database for Arabidopsis proteins</a>.
</dd>
<dt class=g><a class='glossary' name="TargetP_Prediction">TargetP Prediction</a></dt>
<dd>Predicted cellular localization of the protein in question based on the presence of a characteristic sequence motif.  For more information, refer to the TargetP web site at: <a href='http://www.cbs.dtu.dk/services/TargetP/'>www.cbs.dtu.dk/services/TargetP/</a>.
</dd>
<dt class=g><a class='glossary' name="TargetP_Reliability">TargetP Reliability</a></dt>
<dd>Level of certainty in the TargetP predicted localization assignment.  Values range from 1 to 5.  Low values reflect a high level of certainty.  For more information, refer to:<a href='http://www.cbs.dtu.dk/services/TargetP/'>www.cbs.dtu.dk/services/TargetP/</a>.
</dd>
<dt class=g><a class='glossary' name="TargetP_Scores">TargetP Scores</a></dt>
<dd>The highest score indicates the most likely cellular localization of the protein.  For more information, refer to: <a href='http://www.cbs.dtu.dk/services/TargetP/'>www.cbs.dtu.dk/services/TargetP/</a>.
</dd>
<dt class=g><a class='glossary' name="mRNA_Expression_Data">mRNA Expression Data</a></dt>
<dd>Provides convenient links to community expression data for this gene.  Refer also to links provided on the TAIR gene page.
</dd>

</dl><br>
<div class=title>Terms in SeedGenes Mutant Profile</div>
<dl>
<dt class=g><a class='glossary' name="Allele_Symbol">Allele Symbol</a></dt>
<dd>Standard genetic symbol for mutant allele.  The "<i>1S</i>" in "<i>abc-1S</i>" refers to the first Syngenta allele identified for an established locus with an unknown number of other mutant alleles available throughout the community.
</dd>
<dt class=g><a class='glossary' name="Mutant_Class">Mutant Class</a></dt>
<dd>Mutant alleles classified as "Embryo Defective" give an embryo-defective phenotype characterized by a morphological defect in seed development.  Those classified as "Seed Pigment" result in a defect in seed pigmentation (albino, pale green, fusca) but not in morphogenesis.  With the "50% Defective Seeds" class, siliques produce ~50% mutant seeds following self-pollination of heterozygotes, regardless of pollen genotype.
</dd>
<dt class=g><a class='glossary' name="Source_of_Mutant">Source of Mutant</a></dt>
<dd>Original source of mutant seeds and laboratory that provided information on the seed phenotype.
</dd>
<dt class=g><a class='glossary' name="Mutant_Line_Number">Mutant Line Number</a></dt>
<dd>Laboratory number or name used to identify the mutant allele presented.
</dd>
<dt class=g><a class='glossary' name="Ecotype">Ecotype</a></dt>
<dd>Name of the <i>Arabidopsis</i> ecotype in which the mutation was generated.
</dd>
<dt class=g><a class='glossary' name="Genetic_Background">Genetic Background</a></dt>
<dd>The genetic background is assumed to be wild type unless otherwise designated.  The <i>quartet (qrt)</i> mutation present in some lines results in the appearance of clusters of four viable pollen grains, all derived from a single meiosis, that fail to separate during development [<i>Preuss et al. </i>(1994) Science 264: 1458-1460].
</dd>
<dt class=g><a class='glossary' name="ABRC_Stock">ABRC Stock</a></dt>
<dd>Refers to seed stock number from the Arabidopsis Biological Resource Center at Ohio State University, which can be accessed through TAIR.
</dd>
<dt class=g><a class='glossary' name="Mutagen_Treatment">Mutagen Treatment</a></dt>
<dd>Mutagen used to induce the mutation present in the mutant allele.
</dd>
<dt class=g><a class='glossary' name="Insertion_Mutant">Insertion Mutant</a></dt>
<dd>Tagged mutants that contain an inserted piece of foreign DNA responsible for the mutant phenotype are designated with a "Yes".  Clicking on <b><u>Details</u></b> links to additional information on the insertion mutant allele.
</dd>
<dt class=g><a class='glossary' name="Flanking_Sequence">Flanking Sequence</a></dt>
<dd>Click on these links to obtain additional details on flanking plant sequences adjacent to T-DNA inserts in tagged mutants from the Syngenta, Salk, and INRA/Genoplante collections of insertion lines.  Click on "Help" for instructions on how to interpret the information presented.  
</dd>
<dt class=g><a class='glossary' name="Location_of_Mutation">Location of Mutation</a></dt>
<dd>Apparent location of the T-DNA insertion, transposon insertion, deletion, and/or altered nucleotide sequence within and/or near the gene of interest.  Note that some inconsistencies may exist in the numbering of introns and exons when the 5' UTR includes an intron.
</dd>
<dt class=g><a class='glossary' name="Terminal_Phenotype_m">Terminal Phenotype</a></dt>
<dd>Characteristic terminal stage of development reached by mutant embryos based on examination of at least 100 immature mutant seeds under a dissecting microscope.  Clicking on <b><u>Details</u></b> links to additional information and drawings of phenotypic subclasses.  The six phenotypic classes used are: preglobular, globular, transition, cotyledon, fusca, and pigment defective.  The assigned class corresponds to the one most frequently observed or the most developmentally advanced class found in at least 10% of the seeds examined.  Seeds classified as preglobular often contained an early globular embryo too small to be seen upon seed dissection.
</dd>
<dt class=g><a class='glossary' name="Nomarski_Phenotype">Nomarski Phenotype</a></dt>
<dd>Characteristic terminal stage of development reached by mutant embryos too small to be observed with a dissecting microscope.  Mutant seeds were cleared and examined under a compound light microscope equipped with DIC (Nomarski) optics.  Clicking on <b><u>Images</u></b> links to representative images of cleared mutant seeds.
</dd>
<dt class=g><a class='glossary' name="Allelism_Evidence">Allelism Evidence</a></dt>
<dd>The reference allele was chosen based on seed availability and level of phenotypic and/or molecular characterization.  This allele is not necessarily the preferred or most thoroughly studied allele, but rather the allele that represents an appropriate point of reference for this project.  Evidence for allelism with this reference allele is noted as "Complementation Test" if genetic crosses were performed; "Sequence Identity" if this mutant is known to be disrupted in the same gene as the reference allele based on molecular data, or "Map Location and Phenotype" if that represents the best evidence available.
</dd>
<dt class=g><a class='glossary' name="Database_Release_m">Database Release</a></dt>
<dd>Date this mutant allele was first included in the SeedGenes Project.  
</dd>
<dt class=g><a class='glossary' name="Seed_Color_m">Seed Color</a></dt>
<dd>Characteristic color of mutant seeds prior to desiccation.
</dd>
<dt class=g><a class='glossary' name="Embryo_Color_m">Embryo Color</a></dt>
<dd>Characteristic color of mutant embryos prior to desiccation.
</dd>
<dt class=g><a class='glossary' name="Percent_Mutant_Seeds">Percent Mutant Seeds</a></dt>
<dd>Percent mutant seeds observed following self-pollination of a heterozygous plant.  In most cases, 25 siliques were examined and the number and location of mutant and wild-type seeds within each silique were recorded.
</dd>
<dt class=g><a class='glossary' name="Chi_Square_1">Chi-Square</a></dt>
<dd>Based on 25% mutant seeds expected for a typical recessive mutant and 50% mutant seeds expected for mutants with a female gametophytic pattern of inheritance.
</dd>
<dt class=g><a class='glossary' name="Seeds_Counted">Seeds Counted</a></dt>
<dd>Total number of seeds counted to calculate the percent mutant seeds.
</dd>
<dt class=g><a class='glossary' name="Percent_Top_Half">Percent Top Half</a></dt>
<dd>Percentage of total mutant seeds that are located in the top half of the silique, closest to the original stigma surface.  This value is expected to be 50% if the distribution of mutant seeds along the length of the silique is random.  A significant deviation from this expected random distribution often indicates an effect of the mutant allele on pollen development or pollen-tube growth.
</dd>
<dt class=g><a class='glossary' name="Chi_Square_2">Chi-Square</a></dt>
<dd>Based on 50% of the mutant seeds expected to be located in the top half of the silique.
</dd>
<dt class=g><a class='glossary' name="Average_Seed_Length">Average Seed Length</a></dt>
<dd>The maximal lengths of 100 immature mutant seeds removed from heterozygous siliques prior to desiccation were determined using a stage micrometer placed under a dissecting microscope.  Individual measurements were rounded to the nearest 50 microns.  Averages were rounded to the nearest 10 microns.
</dd>
<dt class=g><a class='glossary' name="Standard_Deviation_seed">Standard Deviation</a></dt>
<dd>Calculated for the seed lengths noted above.  Rounded to the nearest 10 microns.
</dd>
<dt class=g><a class='glossary' name="Average_Embryo_Length">Average Embryo Length</a></dt>
<dd>The maximal lengths of 100 embryos removed from mutant seeds prior to desiccation were determined using a stage micrometer placed under a dissecting microscope.  Individual measurements were rounded to the nearest 25 microns.  Averages were rounded to the nearest 10 microns.   Embryos that could not be found because they were too small to be visible under the dissecting microscope were assigned a length of 15 microns based on the known size of the zygote and early embryo in wild-type seeds.
</dd>
<dt class=g><a class='glossary' name="Standard_Deviation_Emb">Standard Deviation</a></dt>
<dd>Calculated for the embryo lengths noted above.  Rounded to the nearest 10 microns.
</dd>
<dt class=g><a class='glossary' name="Last_Modified_Mutant">Last Modified</a></dt>
<dd>Date that information was last added to this portion of the SeedGenes Profile.
</dd>

</dl><br>
<div class=title>Terms in Insertion Allele Table</div>
<dl>
<dt class=g><a class='glossary' name="Allele_Symbol_i">Allele Symbol</a></dt>
<dd>Previously <a href='Glossary.html#Allele_Symbol'>defined.</a>
</dd>
<dt class=g><a class='glossary' name="Mutant_Line_Number_i">Mutant Line Number</a></dt>
<dd>Previously  <a href='Glossary.html#Mutant_Line_Number'>defined.</a>
</dd>
<dt class=g><a class='glossary' name="Gene_Identity_i">Gene Identity</a></dt>
<dd>This is the <A class=glossary title='Glossary Lookup' href='Glossary.html#Chromosome_Locus'>Chromosome Locus</a> previously defined.
</dd>
<dt class=g><a class='glossary' name="Predicted_Gene_Length">Predicted Gene Length</a></dt>
<dd>Length of <A class=glossary title='Glossary Lookup' href='Glossary.html#Predicted_Sequences'>Predicted Gene</a>.
</dd>
<dt class=g><a class='glossary' name="Source_of_Mutant_i">Source of Mutant</a></dt>
<dd>Previously  <a href='Glossary.html#Source_of_Mutant'>defined.</a>
</dd>
<dt class=g><a class='glossary' name="Vector_Name">Vector Name</a></dt>
<dd>Name or source of agent used to generate insertion mutant.  Vectors used for Syngenta populations are described in <a href='publications/InsertionalMutagenesis_Genetics.pdf'>McElver <i>et al</i>. (2001) Genetics 159: 1751-1763.</a>
</dd>
<dt class=g><a class='glossary' name="Selection_Agent">Selection Agent</a></dt>
<dd>Herbicide or antibiotic used to select for transgenic plants carrying a functional insert.
</dd>
<dt class=g><a class='glossary' name="Parental_R_S_Class">Parental R:S Class</a></dt>
<dd>Refer to <a href='publications/InsertionalMutagenesis_Genetics.pdf'>McElver <i>et al</i>. (2001) Genetics 159: 1751-1763</a> for a complete definition and discussion of ratios classes for mutants generated at Syngenta.  Parental insertion lines with a "Low" ratio (< 5:1) of resistant (R) to sensitive (S) progeny typically contain a single site of insertion; those with "Moderate" (5.1-25:1) and "High" (> 25:1) ratios contain additional unlinked inserts.
</dd>
<dt class=g><a class='glossary' name="R_S_Tag_Status">R:S - Tag Status</a></dt>
<dd>Ratio of resistant (R) to sensitive (S) seedlings on selection plates.  Limited to parental lines and subfamilies with a low R:S ratio amenable to determining the tag status.  Expected to be 2:1 if a single insertion disrupts an EMB gene required for seed development and 3:1 if a single insertion disrupts a PDE gene required for normal seed pigmentation.  Refer to <a href='publications/InsertionalMutagenesis_Genetics.pdf'>McElver <i>et al</i>. (2001) Genetics 159: 1751-1763</a> for more information on expected ratios.
</dd>
<dt class=g><a class='glossary' name="Seedlings_Scored">Seedlings Scored</a></dt>
<dd>Number of seedlings screened on plates to determine the R:S ratio presented above.
</dd>
<dt class=g><a class='glossary' name="Transplants_Screened">Transplants Screened</a></dt>
<dd>Number of resistant transplants screened at maturity for the presence of the seed mutant phenotype.
</dd>
<dt class=g><a class='glossary' name="WT_Plants_Found">WT Plants Found</a></dt>
<dd>Number of wild-type plants found among resistant transplants scored for tag status.   This number is expected to be 0 if the mutant is tagged and a single insert is responsible for the mutant phenotype.
</dd>
<dt class=g><a class='glossary' name="Recovery_Rank">Recovery Rank</a></dt>
<dd>These ratings reflect the level of confidence that the correct insertion site has been identified.  Refer to <a href='publications/InsertionalMutagenesis_Genetics.pdf'>McElver <i>et al</i>. (2001) Genetics 159: 1751-1763</a> for a complete definition and discussion.  Rank A = Flanking DNA recovered from both sides of the insert and found to identify the same gene region.  Rank B = Flanking DNA recovered from a single side of the insert.  Recovery of the second border either failed or was not attempted.  Rank Cs = Flanking DNA recovered from both sides of an insert associated with deletion or potential disruption of two or more adjacent genes.  Rank C = Flanking DNA recovered from 2 different inserts located at some distance apart on the same chromosome.  The second insert escaped detection with genetic analysis.  Rank C and Cs alleles are included only if supported by results of genetic complementation tests.
</dd>
<dt class=g><a class='glossary' name="Deletion_Length">Deletion Length</a></dt>
<dd>Number of base pairs missing at the insertion site when plant sequences flanking both sides of the insert were compared with the wild-type genomic sequence.
</dd>
<dt class=g><a class='glossary' name="Location">Location</a></dt>
<dd>Apparent location of the insert within or near the gene of interest.  Some locations are noted as "Uncertain" because existing gene models are either questionable or in the process of being updated.
</dd>
<dt class=g><a class='glossary' name="Recovery_Method">Recovery Method</a></dt>
<dd>Method used to recover and identify plant sequences flanking the insertion site.
</dd>
<dt class=g><a class='glossary' name="Consensus_Length">Consensus Length</a></dt>
<dd>Length in base pairs of consensus border sequence from TAIL-PCR products, plasmid rescue inserts, adaptor or direct PCR products, or some combination thereof for Syngenta mutants.  This may differ from the number of base pairs that align with the genome sequence due to the presence of vector sequences at product ends.
</dd>

</dl><br>
<div class=title>Embryo Phenotype Classification System</div>
<dl>
<dt class=g><a class='glossary' name="Embryo_Classification">Embryo Classification</a></dt>
<dd>The terminal phenotypes of mutant embryos were determined by examining mutant seeds under a dissecting microscope.  Numbers correspond to the percentage of mutant embryos assigned to each phenotype class following dissection of 100 mutant seeds.  Subclasses found in more than 5% of the mutant seeds examined are highlighted with a rectangular border.  Some of the rare phenotype classes and subclasses may correspond to spontaneously aborted seeds rather than homozygous mutant seeds. 
</dd>
</dl>
</div>

<div class='footer'>Address comments to David Meinke, Project PI <a href="mailto:meinke@okstate.edu" class='footer_link'>(meinke@okstate.edu)</a></div>
</body></html>
