<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<HTML><HEAD>
<TITLE>SeedGenes Home Page</TITLE>
<link rel="STYLESHEET" href="seedgenes.css" type="text/css">
<link rel="shortcut icon" href="https://www.vbi.vt.edu/images/favicon.ico" />
</HEAD><BODY>
<div class=header><div class=logoPos1>&nbsp;</div><div class=logoPos2>
<a href='http://www.okstate.edu' ><img alt='OSU' SRC='images/osu.gif' border=0></a>&nbsp;
<a href='http://www.syngenta.com'><img alt='Syngenta' SRC='images/syngenta.gif' border=0></a>&nbsp;
<a href='http://www.vbi.vt.edu'><img alt='vbi' SRC='images/vbi.gif' border=0></a>
</div><a href='index.html'><img SRC='images/seedgenes.jpg' width='396' height='76' border=0></a></div>
<div class=menu1><div class=menu2>&nbsp;
</div></div>
<div class=main700>

<P><font size=6><b>W</b></font>elcome to the former SeedGenes Project website, which presents archival information on
genes with essential functions during Arabidopsis seed development. The original SeedGenes
database was last updated in December, 2010. This archival version composed of static web
pages was introduced without further additions or corrections in September, 2018. More than
60% of the embryo-defective (emb) and pigment-defective embryo (pde) mutants listed here
were analyzed in the Meinke laboratory at Oklahoma State University.</p>
<p>
NOTE: Some of the genes included in this dataset were excluded from later publications based
on updated information from the Meinke lab and elsewhere. The original Query functions of the
SeedGenes database are no longer available with this archival version. Critical information on
Syngenta insertion mutants can be accessed from the lists of genes and mutant alleles presented,
and from the linked SeedGenes Profile pages.
</p>
<div class=title>Updated Information on Genes with Mutant Phenotypes</div>
<p>
David Meinke is preparing an updated dataset and comprehensive review of 500 EMB genes in
Arabidopsis to be published in 2019. Other recent datasets from the Meinke lab focus on genes
required for embryo and gametophyte development (<a href='publications/PLoS_One_2011_Reproductive_Lethality.pdf'>PDF</a>), 2,400 genes with a loss-of-function
mutant phenotype of any kind (<a href='publications/Plant_Physiol_2012_Mutant_Phenotype_Dataset.pdf'>PDF</a>), and 200 genes with a dominant mutant phenotype (<a href='publications/TIPS_2012_Dominant_Mutants.pdf'>PDF</a>).
</p>
<div class=title>Access Outdated Information from SeedGenes (December, 2010 Release)</div>

<div style='text-align: center'>
<p>
<form action="AccessPage.html" method="get"><input value='Enter' type='submit'></form>
</div>
<br>

<div class=title>Project Acknowledgments</div>
<p>
This project was made possible through a T-DNA insertional mutagenesis program
initiated by David Patton at Syngenta. Subsequent
funding was provided by the NSF 2010 Program.  Allan Dickerman at the Virginia
Bioinformatics Institute coordinates database and website functions.  
Seeds for mutants described here are available through the
Arabidopsis stock centers. 
<p>
Please cite the following publication for the SeedGenes database when
utilizing information presented here: <p>Meinke D, Muralla R, Sweeney C,
Dickerman A (2008) Identifying essential genes in <i>Arabidopsis thaliana</i>. Trends
Plant Sci.13: 483-491. 
</div>

</body></html>


